xenstoreclient/xenstore.go

```diff
@@ -534,7 +534,6 @@ func (xs *CachedXenStore) Clear() {
 
 func getDevPath() (devPath string, err error) {
 	devPaths := []string{
-		"/proc/xen/xenbus",
```

Comment:
Would better to mentio https://lkml.org/lkml/2016/5/17/196 in this commit message